build.gradle.kts

```
@@ -2,19 +2,92 @@ plugins {
    id("us.ihmc.ihmc-build")
 }
 
+buildscript {
+   dependencies {
+      classpath("us.ihmc:ros2-msg-to-pubsub-generator:1.2.3")
+   }
+}
+
 ihmc {
    group = "us.ihmc"
    version = "0.1"
    vcsUrl = "https://github.com/ihmcrobotics/ihmc-alexander-sdk"
    openSource = true
 
    // Include SDK content as classpath resources
+   configureDependencyResolution()
    resourceDirectory("main", "../../alexander-models")
    resourceDirectory("main", "../../alexander-ros2")
    ihmc.sourceSet("main").resources.exclude("/ihmc_psyonic_ros2")
+   resourceDirectory("main", "../../alexander-ros2/src/main/generated-idl")
+   javaDirectory("main", "../../alexander-ros2/src/main/generated-java")
```

This looks weird to me. Does this not work? 
```suggestion
   resourceDirectory("main", "generated-idl")
   javaDirectory("main", "generated-java")
```

---

That wouldn't work because the build files are in the repo root, not the alexander-ros2 subfolder. Not to say this couldn't be rethought